### ISSUE TITLE
Behaviour of JS callback with fallback disabled

### ISSUE THREAD

**Issue by JasonBarnabe** [gitcoinco/code_fund_ads#1034]
https://github.com/gitcoinco/code_fund_ads#embed-callbacks

What is `evt.detail` set to if there are no paid ads available and I have disabled fallback ads?

(Please document this.)

**Comment by coderberry** [comment]
@hopsoft or @andrewmcodes this should be a quick addition (and needed)

**Comment by andrewmcodes** [comment]
@JasonBarnabe this is the last thing in the embed-callbacks section.

> And in the event that we do not have an available advertiser, you will see:
> 
> `{ "status": "no-advertiser" }`

I recreated your situation and used this markup:

```html
<!DOCTYPE html>
<html>
  <body>
    <!-- BEGIN COPY CODEFUND EMBED -->
    <div id="codefund"></div>
    <script
      src="https://app.codefund.io/properties/###/funder.js"
      async
    ></script>
    <!-- END COPY CODEFUND EMBED -->

    <script>
      window.addEventListener("codefund", function (evt) {
        if (evt.detail.status !== 'ok') {
          console.log(evt.detail); // {status: "no-advertiser"}
          console.log(evt.detail.status); // no-advertiser
        }
      });
    </script>
  </body>
</html>
```

So if if there are no paid ads available and you have disabled fallback ads, `evt.detail` will be `{status: "no-advertiser"}`. 

If the documentation in the README is unclear, I would love some suggestions for how to improve it!

**Comment by JasonBarnabe** [comment]
Thanks for the answer.

The documentation is unclear because "we do not have an available advertiser" is ambiguous. Is a fallback ad an "available advertiser"? If so, does turning off fallback ads make them "unavailable"? Are there scenarios where you'd get "no-advertiser" even if fallback ads are on?

I would suggest rewording to something like:

> And in the event we have no ad to show (no paid ad, and you've turned off fallback ads), you will see:

**Comment by andrewmcodes** [comment]
@JasonBarnabe thanks, I will make an update to the wording. 

No advertiser means there is no available paid or fallback advertiser so I will make sure that it becomes more clear.
